squash/squash/urls.py

```diff
@@ -0,0 +1,24 @@
+"""squash URL Configuration
+
+The `urlpatterns` list routes URLs to views. For more information please see:
+    https://docs.djangoproject.com/en/1.8/topics/http/urls/
+Examples:
+Function views
+    1. Add an import:  from my_app import views
+    2. Add a URL to urlpatterns:  url(r'^$', views.home, name='home')
+Class-based views
+    1. Add an import:  from other_app.views import Home
+    2. Add a URL to urlpatterns:  url(r'^$', Home.as_view(), name='home')
+Including another URLconf
+    1. Add an import:  from blog import urls as blog_urls
+    2. Add a URL to urlpatterns:  url(r'^blog/', include(blog_urls))
+"""
+from django.conf.urls import include, url
+from django.contrib import admin
+from . import views
+
+urlpatterns = [
+    	url(r'^admin/', include(admin.site.urls)),
+	url(r'^(?P<name>[\w./-]+)/$', views.show_page, name='page'),
```

Comment:
This route doesn't necessarily have to be dynamic at this point. So worse come to worse it's specific for each template.